src/utils/relayer.ts

```diff
@@ -132,38 +142,3 @@ export async function loadBlockchain(
     blockchainRedirects,
   } as BlockchainDetails)
 }
-
-// Get blockchain's alias by it's redirect domain
-export async function getBlockchainAliasesByDomain(
```

Comment:
Is there a specific reason why this could not be used, as it is already performing the match on redirects.